melodic/distribution.yaml

```diff
@@ -12177,6 +12177,17 @@ repositories:
       url: https://github.com/gbiggs/rtsprofile.git
       version: master
     status: maintained
+  ruckig:
+    release:
+      tags:
+        release: release/melodic/{package}/{version}
+      url: https://github.com/pantor/ruckig-release.git
+      version: 0.3.3-1
+    source:
+      type: git
+      url: https://github.com/pantor/ruckig.git
+      version: v0.3.3
```

Comment:
A tag is not going to work here; the source needs to point to a branch.  I'll suggest changing this to `main`.

Reply:
As I'm not really familiar with the ROS release process: Can I change that within this PR or do I need to repeat the whole process with `bloom-release`?

Reply:
You can just change it in this PR.  That is actually preferred.

Reply:
Changed that to `master`.